static/js/src/dynamic-forms.js

```diff
@@ -273,6 +283,49 @@
         });
       });
 
+      // Checks additional required fields to see whether a value has been set
+      function validateForm(form) {
+        var fields = form.querySelectorAll("[required]");
+        var validStates = [];
+
+        fields.forEach((field) => {
+          var fieldName = field.getAttribute("name");
+          var inputs = form.querySelectorAll(`[name="${fieldName}"]`);
+          var validationMessage = document.querySelector(
+            `.js-validation-${fieldName}`
+          );
+          var inputValid = false;
+
+          inputs.forEach((input) => {
+            if (input.type === "checkbox" && input.checked) {
+              inputValid = true;
+            }
+
+            if (input.type === "radio" && input.checked) {
+              inputValid = true;
+            }
+
+            if (input.type === "text" && input.value) {
+              inputValid = true;
+            }
```

Comment:
It seems only the textarea is required in this form but is this for future use? just wondering!

Reply:
ah yes - yeah, this function should make it possible for us to add required fields to other forms in future.